src/app/side-nav/side-nav.component.scss

```diff
@@ -0,0 +1,25 @@
+
+mat-sidenav-container{
+  height: calc(205vh - 105vh);
+}
+mat-sidenav{
+  width: 300px;
+  background-color: #8341c1;
+  text-align: center;
+}
+a{
+  color: white;
+  text-decoration: none;
+}
+
+label{
```

Comment:
I think you don't have label tag in this template 
Also check tag nesting, more Scss style